DateRangeBox: add new API description (#5026)
DevExpress/devextreme-documentation: api-reference/10 UI Components/DateBoxBase/1 Configuration/min.md

---
---
##### shortDescription

> The minimum date that can be selected within the UI component.

Reviewer: ```suggestion
The earliest date that can be selected in the UI component.
```
